modular_skyrat/modules/modular_clothing/armor/armor.dm

```diff
@@ -0,0 +1,72 @@
+//TECHARMOR
+/obj/item/clothing/head/helmet/space/hardsuit/security_armor //No, this isn't spaceproof. Yes, the sprite reflects that this is not spaceproof properly. I made it a hardsuit subtype to make things easier and more readable.
+	name = "type I techhelmet"
+	desc = "A specialized exoskeleton armor helmet built into a suit of armor; offers decent protection, and comes with a flash-resistant HUD visor and headlamp."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/hats.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	flash_protect = FLASH_PROTECTION_FLASH //The visor protects from flashes, but you're still going to go blind if you try to weld something.
+	icon_state = "hardsuit-secexo"
+	icon_state = "hardsuit0-secexo"
+	inhand_icon_state = "hardsuit0-secexo"
+	hardsuit_type = "secexo"
+	armor = list("melee" = 40, "bullet" = 30, "laser" = 30,"energy" = 10, "bomb" = 25, "bio" = 50, "rad" = 0, "fire" = 55, "acid" = 70)
+	clothing_flags = THICKMATERIAL | BLOCK_GAS_SMOKE_EFFECT | MASKINTERNALS
```

Comment:
Are helmets being usable for internals shared by other environmental suits or is that strictly on these armors? I know it was a Citadel change but that isn't the case on /tg/. It might be a good idea to take that flag out until such time that someone modularizes the other suits if they want that change or someone proposes it on /tg/.

Reply:
This was a citadel change, and personally I'd like to keep it around since it doesn't seem like it would cause many issues. If it does though, ill remove it later.